src/node_lib.h

```diff
@@ -80,9 +80,12 @@ bool eventLoopIsRunning();
  * not called.
  * @param program_name The name for the Node.js application.
  * @param node_args List of arguments for the Node.js engine.
+ * @param allow_repl Controls wether the node.js REPL gets spawned when no
+ * script is given on initialization.
```

Comment:
Is this correct? Isn't the REPL also spawned when there is a script given but `stdin` is an interactive terminal?

Reply:
The way I understood, stdin only is an interactive terminal when no file is given, but also i may just also be completely misunderstanding the way this works... I mean, with the usual nodejs usage, where you give a script as a CLI argument, stdin still is the terminal, right? But the REPL is not spawned, which is why I gave that description in the doc. Feel free to correct me though...